utils/mp4AudioFormats.ts

```diff
@@ -47,19 +45,17 @@ export function hasAacSupport(videoTestElement: HTMLVideoElement): boolean {
 }
 
 /**
- *
- *
- * @returns
+ * @returns {boolean} If browser has MP2 support
  */
-function hasMp2AudioSupport() {
+function hasMp2AudioSupport(): boolean {
   return browserDetector.isTv();
 }
 
 /**
+ * Function for Determining DTS audio support
```

Comment:
```suggestion
 * Determines if the current platform has DTS support
```

No need to specify it's a function (it's obvious :yum: ) and DTS is always audio.

Reply:
I've also removed this description since the @returns describes what the function does